frontend/javascripts/viewer/view/version_entry.tsx

```diff
@@ -374,35 +381,99 @@ const descriptionFns: Record<
     icon: <EditOutlined />,
   }),
   updateSkeletonTracing: (): Description => updateTracingDescription,
   updateVolumeTracing: (): Description => updateTracingDescription,
-  addLayerToAnnotation: (action: AddLayerToAnnotationUpdateAction): Description => ({
+  updateActiveSegmentId: (
+    action: AsServerAction<UpdateActiveSegmentIdUpdateAction>,
+  ): Description => {
+    return {
+      description: `Activated segment id ${action.value.activeSegmentId}`,
+      icon: <EditOutlined />,
+    };
+  },
+  addLayerToAnnotation: (
+    action: AsServerAction<AddLayerToAnnotationUpdateAction>,
+  ): Description => ({
     description: `Added the layer ${action.value.layerParameters.name} to the annotation.`,
     icon: <PlusOutlined />,
   }),
-  deleteLayerFromAnnotation: (action: DeleteAnnotationLayerUpdateAction): Description => ({
+  deleteLayerFromAnnotation: (
+    action: AsServerAction<DeleteAnnotationLayerUpdateAction>,
+  ): Description => ({
     description: `Deleted the layer with id ${action.value.layerName} (${action.value.tracingId}) from the annotation.`,
     icon: <DeleteOutlined />,
   }),
-  updateLayerMetadata: (action: UpdateAnnotationLayerNameUpdateAction): Description => ({
+  updateLayerMetadata: (
+    action: AsServerAction<UpdateAnnotationLayerNameUpdateAction>,
+  ): Description => ({
     description: `Updated the name of the layer with id ${action.value.tracingId} to ${action.value.layerName}.`,
     icon: <EditOutlined />,
   }),
-  updateMetadataOfAnnotation: (action: UpdateMetadataOfAnnotationUpdateAction): Description => {
+  updateMetadataOfAnnotation: (
+    action: AsServerAction<UpdateMetadataOfAnnotationUpdateAction>,
+  ): Description => {
     return {
       description: `Updated the description of the annotation to: ${action.value.description.slice(0, 100) || ""}`,
       icon: <EditOutlined />,
     };
   },
+  updateActiveNode: (action: AsServerAction<UpdateActiveNodeUpdateAction>): Description => {
+    return {
+      description: `Updated the active node id to ${action.value.activeNode}`,
+      icon: <EditOutlined />,
+    };
+  },
+  updateCamera: (_action: AsServerAction<UpdateCameraAnnotationAction>): Description => {
+    return {
+      description: "Updated the camera position",
+      icon: <VideoCameraOutlined />,
+    };
+  },
+  updateLargestSegmentId: (
+    action: AsServerAction<UpdateLargestSegmentIdVolumeAction>,
+  ): Description => {
+    return {
+      description: `Set largest segment id to ${action.value.largestSegmentId}`,
+      icon: <NumberOutlined />,
+    };
+  },
+  updateSegmentGroupsExpandedState: (
+    action: AsServerAction<UpdateSegmentGroupsExpandedStateUpdateAction>,
+  ): Description => {
+    return {
+      description: `${action.value.areExpanded ? "Expanded" : "Collapsed"} some segment groups.`,
+      icon: <FolderOpenOutlined />,
+    };
+  },
+  updateTreeGroupsExpandedState: (
+    action: AsServerAction<UpdateSegmentGroupsExpandedStateUpdateAction>,
```

Comment:
```suggestion
    action: AsServerAction<UpdateTreeGroupsExpandedStateSkeletonAction>,
```

Needs to be imported as well.